packages/ui/src/Checkbox.module.scss

```
@@ -2,13 +2,17 @@
 @use '../styles/helpers' as h;
 
 .wrapper {
-  display: inline-flex;
+  display: flex;
   flex-direction: row;
-  user-select: none;
   cursor: pointer;
 
   @include c.typographyBodySmall;
 
+  // hide native checkbox
+  input {
+    @include h.visuallyHidden;
+  }
+
```

Don't we need it since we changed the display value from `inline-flex` to `flex`?
```suggestion

.errorContainer {
  margin-top: c.$grid;
}
```
I think it is the last comment. Feel free to merge the PR in once it is resolved.